commit 21d72710885034e1c7690ac6bb9c1304c73a4061
Author: Matthew Booth <mbooth@redhat.com>
Date:   Tue Oct 16 14:33:34 2012 +0100

    Fix regression introduced by bcd46673ca3c4259ff23c9c9954adb80f77ceacc
    
    Incorrect use of format_pools meant that all libvirt volumes were marked as
    raw.
    (cherry picked from commit 12964120db69c1328cda1104755218067b1f0f72)

diff --git a/lib/Sys/VirtConvert/Connection/LibVirtSource.pm b/lib/Sys/VirtConvert/Connection/LibVirtSource.pm
index b9ebe6f..3829ca6 100644
--- a/lib/Sys/VirtConvert/Connection/LibVirtSource.pm
+++ b/lib/Sys/VirtConvert/Connection/LibVirtSource.pm
@@ -181,7 +181,7 @@ sub get_volume
             $pooltype = $pooltype->getNodeValue();
 
             $format = 'raw'
-                unless $Sys::VirtConvert::Libvirt::format_pools{$pooltype};
+                unless $Sys::VirtConvert::Connection::LibVirt::format_pools{$pooltype};
         } else { # Should be impossible
             logmsg WARN, __x('Pool XML has no type attribute: {xml}',
                              xml => $pool->get_xml_description());
diff --git a/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm b/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm
index 85b4b1a..cf9c78b 100644
--- a/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm
+++ b/lib/Sys/VirtConvert/Connection/LibVirtTarget.pm
@@ -174,7 +174,7 @@ sub create_volume
     if (defined($pooltype)) {
         $pooltype = $pooltype->getNodeValue();
 
-        if ($Sys::VirtConvert::Libvirt::format_pools{$pooltype}) {
+        if ($Sys::VirtConvert::Connection::LibVirt::format_pools{$pooltype}) {
             $vol_xml = $vol_xml_format;
         } else {
             $vol_xml = $vol_xml_noformat;
